include/aie/Targets/cdo_driver.h

```diff
@@ -0,0 +1,52 @@
+// 67d7842dbbe25473c3c32b93c0da8047785f30d78e8a024de1b57352245f9689
+/* cdo_driver.h  */
+#ifndef _CDO_DRIVER_H_
+#define _CDO_DRIVER_H_
```

Comment:
It seems weird to have these Decls in our code.  Where did this come from?

Reply:
No clue - I wasn't sure if they were (or weren't) used before I started but now I can see it's safe to delete all except `byte_ordering`.